superset-frontend/src/views/CRUD/alert/components/AlertStatusIcon.tsx

```diff
@@ -16,34 +16,31 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { styled, t } from '@superset-ui/core';
+import { t, supersetTheme } from '@superset-ui/core';
```

Comment:
let's try not to use the supersetTheme directly, but get our variables via the ThemeProvider. You can use `withTheme` (HOC) or `useTheme` (hook) in most cases.